
(sonicstylegan-pt) C:\Users\James Gibbons-MacGre>cd lucid-sonic-dreams

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 torchaudio==0.8.1 -f https://download.pytorch.org/whl/torch_stable.html
Looking in links: https://download.pytorch.org/whl/torch_stable.html
Requirement already satisfied: torch==1.8.1+cu111 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (1.8.1+cu111)
Requirement already satisfied: torchvision==0.9.1+cu111 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (0.9.1+cu111)
Requirement already satisfied: torchaudio==0.8.1 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (0.8.1)
Requirement already satisfied: numpy in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from torch==1.8.1+cu111) (1.22.4)
Requirement already satisfied: typing-extensions in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from torch==1.8.1+cu111) (4.2.0)
Requirement already satisfied: pillow>=4.1.1 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from torchvision==0.9.1+cu111) (9.1.1)

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>pip install click requests ninja imageio imageio-ffmpeg tqdm psutil scipy pyspng
Requirement already satisfied: click in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (8.1.3)
Requirement already satisfied: requests in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (2.28.0)
Requirement already satisfied: ninja in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (1.10.2.3)
Requirement already satisfied: imageio in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (2.19.3)
Requirement already satisfied: imageio-ffmpeg in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (0.4.7)
Requirement already satisfied: tqdm in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (4.64.0)
Requirement already satisfied: psutil in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (5.9.1)
Requirement already satisfied: scipy in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (1.8.1)
Requirement already satisfied: pyspng in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (0.1.0)
Requirement already satisfied: colorama in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from click) (0.4.5)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from requests) (2022.6.15)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from requests) (1.26.9)
Requirement already satisfied: idna<4,>=2.5 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from requests) (3.3)
Requirement already satisfied: charset-normalizer~=2.0.0 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from requests) (2.0.12)
Requirement already satisfied: pillow>=8.3.2 in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from imageio) (9.1.1)
Requirement already satisfied: numpy in c:\users\james gibbons-macgre\.conda\envs\sonicstylegan-pt\lib\site-packages (from imageio) (1.22.4)

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
Preparing style...
Loading networks from C:/Users/James Gibbons-MacGre/lucid-sonic-dreams/network-snapshot-000017.pkl...
Traceback (most recent call last):
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\RUNTEST1.py", line 7, in <module>
    L.hallucinate(L.hallucinate(file_name = 'printsVidTEST1.mp4',
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\lucidsonicdreams\main.py", line 708, in hallucinate
    self.stylegan_init()
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\lucidsonicdreams\main.py", line 178, in stylegan_init
    with self.dnnlib.util.open_url(weights_file) as f:
  File "C:\Users\James Gibbons-MacGre\.conda\envs\sonicstylegan-pt\lib\site-packages\dnnlib\util.py", line 347, in open_url
    assert is_url(url)
AssertionError

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
Preparing style...
Loading networks from C:/Users/James Gibbons-MacGre/lucid-sonic-dreams/network-snapshot-000017.pkl...
Downloading C:/Users/James Gibbons-MacGre/lucid-sonic-dreams/network-snapshot-000017.pkl ............ failed
Traceback (most recent call last):
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\RUNTEST1.py", line 7, in <module>
    L.hallucinate(L.hallucinate(file_name = 'printsVidTEST1.mp4',
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\lucidsonicdreams\main.py", line 708, in hallucinate
    self.stylegan_init()
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\lucidsonicdreams\main.py", line 178, in stylegan_init
    with self.dnnlib.util.open_url(weights_file) as f:
  File "C:\Users\James Gibbons-MacGre\.conda\envs\sonicstylegan-pt\lib\site-packages\dnnlib\util.py", line 365, in open_url
    with session.get(url) as res:
  File "C:\Users\James Gibbons-MacGre\.conda\envs\sonicstylegan-pt\lib\site-packages\requests\sessions.py", line 600, in get
    return self.request("GET", url, **kwargs)
  File "C:\Users\James Gibbons-MacGre\.conda\envs\sonicstylegan-pt\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\James Gibbons-MacGre\.conda\envs\sonicstylegan-pt\lib\site-packages\requests\sessions.py", line 695, in send
    adapter = self.get_adapter(url=request.url)
  File "C:\Users\James Gibbons-MacGre\.conda\envs\sonicstylegan-pt\lib\site-packages\requests\sessions.py", line 792, in get_adapter
    raise InvalidSchema(f"No connection adapters were found for {url!r}")
requests.exceptions.InvalidSchema: No connection adapters were found for 'C:/Users/James Gibbons-MacGre/lucid-sonic-dreams/network-snapshot-000017.pkl'

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
Preparing style...
Style not valid. Call show_styles() to see all valid styles, or use your own .pkl file.

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\RUNTEST1.py", line 3
    gdown --id 1-ZNBMR3-3yBC80vxuoq1_JA8N9UQmbj0
               ^
SyntaxError: invalid syntax

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\RUNTEST1.py", line 3
    !gdown --id 1-ZNBMR3-3yBC80vxuoq1_JA8N9UQmbj0
    ^
SyntaxError: invalid syntax

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
Traceback (most recent call last):
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\RUNTEST1.py", line 5, in <module>
    gdown.download(id=id, output=output, quiet=False)
NameError: name 'gdown' is not defined

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\RUNTEST1.py", line 2
    pip install gdown
        ^
SyntaxError: invalid syntax

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>conda install gdown
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\Users\James Gibbons-MacGre\.conda\envs\sonicstylegan-pt

  added / updated specs:
    - gdown


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    ca-certificates-2022.4.26  |       haa95532_0         124 KB
    ------------------------------------------------------------
                                           Total:         124 KB

The following NEW packages will be INSTALLED:

  pip                pkgs/main/win-64::pip-21.2.4-py39haa95532_0

The following packages will be SUPERSEDED by a higher-priority channel:

  ca-certificates    conda-forge::ca-certificates-2022.6.1~ --> pkgs/main::ca-certificates-2022.4.26-haa95532_0
  certifi            conda-forge::certifi-2022.6.15-py39hc~ --> pkgs/main::certifi-2022.6.15-py39haa95532_0
  openssl            conda-forge::openssl-1.1.1p-h8ffe710_0 --> pkgs/main::openssl-1.1.1o-h2bbff1b_0


Proceed ([y]/n)? y


Downloading and Extracting Packages
ca-certificates-2022 | 124 KB    | ############################################################################ | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
Traceback (most recent call last):
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\RUNTEST1.py", line 6, in <module>
    gdown.download(id=id, output=output, quiet=False)
NameError: name 'gdown' is not defined

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>python RUNTEST1.py
Downloading...
From: https://drive.google.com/file/d/1-ZNBMR3-3yBC80vxuoq1_JA8N9UQmbj0/view?usp=sharing
To: C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\network-snapshot-000029.pkl
68.9kB [00:00, 986kB/s]
Preparing style...
Loading networks from network-snapshot-000029.pkl...
Traceback (most recent call last):
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\RUNTEST1.py", line 13, in <module>
    L.hallucinate(L.hallucinate(file_name = 'printsVidTEST1.mp4',
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\lucidsonicdreams\main.py", line 708, in hallucinate
    self.stylegan_init()
  File "C:\Users\James Gibbons-MacGre\lucid-sonic-dreams\lucidsonicdreams\main.py", line 178, in stylegan_init
    with self.dnnlib.util.open_url(weights_file) as f:
  File "C:\Users\James Gibbons-MacGre\.conda\envs\sonicstylegan-pt\lib\site-packages\dnnlib\util.py", line 347, in open_url
    assert is_url(url)
AssertionError

(sonicstylegan-pt) C:\Users\James Gibbons-MacGre\lucid-sonic-dreams>




